synapse/storage/events.py

```diff
@@ -178,7 +195,7 @@ def get_events(self, event_ids, check_redacted=True,
 
     @log_function
     def _persist_event_txn(self, txn, event, context,
-                           is_new_state=True, current_state=None):
+                           is_new_state, current_state):
```

Comment:
Again, this wasn't strictly relevant to the PR, but there isn't much point in giving defaults in an internal function called from exactly one place.